pkg/cnservice/server.go

```diff
@@ -87,15 +94,16 @@ func (s *service) Close() error {
 }
 
 func (s *service) acquireMessage() morpc.Message {
-	return s.pool.Get().(*pipeline.Message)
+	return s.responsePool.Get().(*pipeline.Message)
+}
+
+func defaultRequestHandler(ctx context.Context, message morpc.Message, cs morpc.ClientSession) error {
+	return nil
 }
 
-/*
-func (s *service) releaseMessage(msg *pipeline.Message) {
-	msg.Reset()
-	s.pool.Put(msg)
+func (s *service) handleRequest(ctx context.Context, req morpc.Message, _ uint64, cs morpc.ClientSession) error {
```

Comment:
SCA Error, unused

Reply:
it's really unused now. I will comment it.